### ISSUE TITLE
gather workflows for end-to-end testing

### ISSUE THREAD

**Issue by mr-c** [WorkflowConversion/CTDConverter#55]
@timosachsenberg , Oliver said you might have some Galaxy workflows that used the output of CTDConverter?

**Comment by timosachsenberg** [comment]
Not sure. AFAIK, we generate our galaxy node wrappers using CTD and we have some galaxy workflows. Would this help?

**Comment by bgruening** [comment]
GTN has a few workflows included that worked last time we used them. However, those are training workflows.

**Comment by mr-c** [comment]
> Not sure. AFAIK, we generate our galaxy node wrappers using CTD and we have some galaxy workflows. Would this help?

Yes, any workflow that uses CTD-derived tool descriptions would be useful!

**Comment by bernt-matthias** [comment]
Yep. All OpenMS tools for version > 2.5 are derived from CTD (99.9% automatically with a few [2] exceptions where patches are used). The workflows @timosachsenberg is working on should use the latest version of these tools.

**Comment by mr-c** [comment]
@timosachsenberg can you link me to your workflows? Is there (small) testing data?

**Comment by mr-c** [comment]
Metabolite_Adduct _Grouping

WF: https://usegalaxy.eu/u/annalefarova/w/metaboliteadduct-grouping
History:  https://usegalaxy.eu/u/annalefarova/h/metaboliteadductgrouping
Data: 
https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Tutorials/Data/latest/Example_Data/Metabolomics/

Proteomics_LFQ

WF: https://usegalaxy.eu/u/annalefarova/w/proteomicslfq
Data: 1) 
https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Tutorials/Data/latest/Example_Data/Labelfree/datasets/
2) 
https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Tutorials/Data/latest/Example_Data/Labelfree/databases/s_pyo_sf370_potato_human_target_decoy_with_contaminants.fasta
